desktop/containerd/index.md

```diff
@@ -32,186 +65,89 @@ images** checkbox.
-  hubproxy.docker.internal:5000
-  127.0.0.0/8
- Live Restore Enabled: false
-
-```
-
-### Run a simple container
-
-```console
-$ docker run --rm hello-world
-Unable to find image 'hello-world:latest' locally
-7d246653d051: Download complete
-432f982638b3: Download complete
-46331d942d63: Download complete
-7050e35b49f5: Downloading [>                                                  ]       0B/3.208kB
-
-Hello from Docker!
-This message shows that your installation appears to be working correctly.
-...
-
-```
-
-## Run the container
+## Building multi-platform images
 
-Run the container specifying port settings:
+The term multi-platform image refers to a bundle of images that can run on different architectures.
+Out of the box, the default builder for Docker Desktop doesn't support building multi-platform images.
 
 ```console
-$ docker run -p 8080:80 -d nginx
-
-Unable to find image 'nginx:latest' locally
-b95a99feebf7: Download complete
-91d5b6827ff7: Download complete
-fc5ec3f147e4: Download complete
-5b1423465504: Download complete
-1cdde8b981f2: Download complete
-6c0b05f215c0: Download complete
-004f1937a10a: Download complete
-fd61d71c75fe: Download complete
-717bf61a04cf: Download complete
-8b6a7e0df8edbe91dfa77716a5e84ca28348f44f545a0d34c70a8987c56e63e0
-
+$ docker buildx ls | grep "DRIVER\|*"
+NAME/NODE       DRIVER/ENDPOINT             STATUS  BUILDKIT PLATFORMS
+default *       docker
+$ docker buildx build --platform=linux/amd64,linux/arm64 .
+[+] Building 0.0s (0/0)
+ERROR: multiple platforms feature is currently not supported for docker driver. Please switch to a different driver (eg. "docker buildx create --use")
 ```
 
-Confirm the Nginx container is running:
+Normally, building multi-platform images requires you to create a new builder,
+using a driver that supports multi-platform builds.
+But even then, you can't load the multi-platform images to your local image store.
 
 ```console
-$ docker ps
-CONTAINER ID   IMAGE     COMMAND                  CREATED         STATUS         PORTS                  NAMES
-93b4d60dfd08   nginx     "/docker-entrypoint.…"   3 seconds ago   Up 3 seconds   0.0.0.0:8080->80/tcp   stoic_mccarthy
+$ docker buildx create --bootstrap
+[+] Building 2.4s (1/1) FINISHED
+ => [internal] booting buildkit
+ => => pulling image moby/buildkit:buildx-stable-1
+ => => creating container buildx_buildkit_objective_blackburn0
+objective_blackburn
+$ docker buildx build --quiet \
+  --platform=linux/amd64,linux/arm64 \
+  --builder=objective_blackburn \
+  --load .
+ERROR: docker exporter does not currently support exporting manifest lists
 ```
 
-You can also check from the browser that Nginx is running:
-
-![Containerd setting in Docker Desktop](../images/containerd_feature_nginx.png){:width="750px"}
-
-## Building multi-platform images
-
-Sample Dockerfile:
+Enabling the containerd image store lets you build, and load, multi-platform images
+to your local image store, all while using the default builder.
```

Comment:
Is this only valid for a unreleased version of Docker? enabling `conainerd` under "Features in development" in Docker Desktop 4.18.0 and then following the above steps still results in the same error as without `containerd`. Logs:


```log
2023-04-24 11:03:31 time="2023-04-24T09:03:31Z" level=info msg="auto snapshotter: using overlayfs"
2023-04-24 11:03:31 time="2023-04-24T09:03:31Z" level=warning msg="using host network as the default"
2023-04-24 11:03:31 time="2023-04-24T09:03:31Z" level=info msg="found worker \"ohemlvts8rh4pro3djyifyjuz\", labels=map[org.mobyproject.buildkit.worker.executor:oci org.mobyproject.buildkit.worker.hostname:abe2a757ca2d org.mobyproject.buildkit.worker.network:host org.mobyproject.buildkit.worker.oci.process-mode:sandbox org.mobyproject.buildkit.worker.selinux.enabled:false org.mobyproject.buildkit.worker.snapshotter:overlayfs], platforms=[linux/arm64 linux/amd64 linux/amd64/v2 linux/riscv64 linux/ppc64le linux/s390x linux/386 linux/mips64le linux/mips64 linux/arm/v7 linux/arm/v6]"
2023-04-24 11:03:31 time="2023-04-24T09:03:31Z" level=warning msg="skipping containerd worker, as \"/run/containerd/containerd.sock\" does not exist"
2023-04-24 11:03:31 time="2023-04-24T09:03:31Z" level=info msg="found 1 workers, default=\"ohemlvts8rh4pro3djyifyjuz\""
2023-04-24 11:03:31 time="2023-04-24T09:03:31Z" level=warning msg="currently, only the default worker can be used."
2023-04-24 11:03:31 time="2023-04-24T09:03:31Z" level=info msg="running server on /run/buildkit/buildkitd.sock"
```

Reply:
No, this is true for Docker Desktop 4.18 as well. What's your `docker info` output?

Reply:
```
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc., v0.10.4)
  compose: Docker Compose (Docker Inc., v2.12.2)
  dev: Docker Dev Environments (Docker Inc., v0.1.0)
  extension: Manages Docker extensions (Docker Inc., v0.2.19)
  init: Creates Docker-related starter files for your project (Docker Inc., v0.1.0-beta.2)
  sbom: View the packaged-based Software Bill Of Materials (SBOM) for an image (Anchore Inc., 0.6.0)
  scan: Docker Scan (Docker Inc., v0.25.0)
  scout: Command line tool for Docker Scout (Docker Inc., v0.9.0)

Server:
 Containers: 2
  Running: 2
  Paused: 0
  Stopped: 0
 Images: 1
 Server Version: 22.06.0-beta.0-926-g914b02ebaf.m
 Storage Driver: stargz
  driver-type: io.containerd.snapshotter.v1
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 2456e983eb9e37e47538f59ea18f2043c9a73640
 runc version: v1.1.4-0-g5fd4c4d
 init version: de40ad0
 Security Options:
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 5.15.49-linuxkit
 Operating System: Docker Desktop
 OSType: linux
 Architecture: aarch64
 CPUs: 10
 Total Memory: 15.61GiB
 Name: docker-desktop
 ID: 9df56d26-7dbe-46c2-a946-427b0766499b
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 HTTP Proxy: http.docker.internal:3128
 HTTPS Proxy: http.docker.internal:3128
 No Proxy: hubproxy.docker.internal
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  hubproxy.docker.internal:5555
  127.0.0.0/8
 Live Restore Enabled: false
```

Reply:
@NiklasBr could you open an issue over at https://github.com/docker/for-mac/ please?

And give us as much info as you can and we'll try and see what the issue is

Reply:
If you are using DD for linux then you can create an issue here: https://github.com/docker/desktop-linux/issues

Reply:
@rumpl Issue opened: https://github.com/docker/for-mac/issues/6816